src/k8s-extension/azext_k8s_extension/partner_extensions/AzureMLKubernetes.py

```diff
@@ -171,12 +171,42 @@ def __validate_config(self, configuration_settings, configuration_protected_sett
                 "for Machine Learning training or inference by specifying "
                 f"'--configuration-settings {self.ENABLE_TRAINING}=true' or '--configuration-settings {self.ENABLE_INFERENCE}=true'")
 
+        self.__validate_scoring_fe_settings(configuration_settings, configuration_protected_settings)
+
         configuration_settings[self.ENABLE_TRAINING] = configuration_settings.get(self.ENABLE_TRAINING, enable_training)
         configuration_settings[self.ENABLE_INFERENCE] = configuration_settings.get(
             self.ENABLE_INFERENCE, enable_inference)
         configuration_protected_settings.pop(self.ENABLE_TRAINING, None)
         configuration_protected_settings.pop(self.ENABLE_INFERENCE, None)
 
+    def __validate_scoring_fe_settings(self, configuration_settings, configuration_protected_settings):
+        clusterPurpose = _get_value_from_config_protected_config(
+            'clusterPurpose', configuration_settings, configuration_protected_settings)
+        if clusterPurpose and clusterPurpose not in ["DevTest", "FastProd"]:
```

Comment:
there is alway another value "test"

Reply:
fixed in helm chart. Should be either DevTest or FastProd